pandas/tests/internals/test_internals.py

```diff
@@ -26,9 +24,6 @@
 from pandas.core.internals import BlockManager, SingleBlockManager, make_block
 import pandas.util.testing as tm
 
-# in 3.6.1 a c-api slicing function changed, see src/compat_helper.h
-PY361 = LooseVersion(sys.version) >= LooseVersion("3.6.1")
```

Comment:
have we dropped 3.6.0?

Reply:
our min is 3.6.1 i think

Reply:
Yes, something is broken in 3.6.0, we were getting this test failures:
```
_______________ TestFrozenNDArray.test_tricky_container_to_bytes _______________
[gw0] linux -- Python 3.6.0 /home/vsts/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.indexes.test_frozen.TestFrozenNDArray object at 0x7fd89d5b6d68>

    def test_tricky_container_to_bytes(self):
>       bytes(self.unicode_container)
E       TypeError: only integer scalar arrays can be converted to a scalar index

pandas/tests/indexes/test_frozen.py:79: TypeError
_______________ TestFrozenNDArray.test_string_methods_dont_fail ________________
[gw0] linux -- Python 3.6.0 /home/vsts/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.indexes.test_frozen.TestFrozenNDArray object at 0x7fd89d037e80>

    def test_string_methods_dont_fail(self):
        repr(self.container)
        str(self.container)
>       bytes(self.container)
E       TypeError: only integer scalar arrays can be converted to a scalar index

pandas/tests/test_base.py:47: TypeError
```
So we require 3.6.1 since #29212